tests/snapshots/parse_file__repeated_datatype_name.bend.snap

```diff
@@ -4,4 +4,8 @@ input_file: tests/golden_tests/parse_file/repeated_datatype_name.bend
 ---
 [4m[1m[31mErrors:[0m
 In tests/golden_tests/parse_file/repeated_datatype_name.bend :
-Found a repeated field 'Expr' in constructor Expr/Plus.
+[1m- information:[0m Found a repeated field 'Expr' in constructor Expr/Plus.
+[1m- expected:[0m field
```

Comment:
It'd be better to not have the "expected" "detected" part here, since it is a field, just not a good one.

Either that or change the message to `unique field` or similar.

Reply:
Changed it to show `information` and `location` without `expected`, what do you think?

Reply:
Looks good